test/endToEnd/repeatedfield/write.py

```diff
@@ -0,0 +1,52 @@
+import repeatedfield_pb2
```

Comment:
I'm happe to see tests along these lines, to make sure we can interoperate with the Python protobuf support. But are these actually run automatically yet?

Reply:
Yes, these tests run automatically using the `[test](https://github.com/Aniket21mathur/Chapel-protobuf/blob/master/test.sh)` shell file that I have written. But I am not sure if that would be the best way to integrate the end to end tests with the main chapel repo.

Reply:
Yes in that situation we will need to use `start_test` to run them, somehow. It is possible to write a `sub_test` program that does anything you like and that is one option for running these tests. Another is writing a Chapel program that is a test that runs the right commands to run these.

Reply:
Yes, I will look into that. Also, we are doing the end testing with Python only, I guess that should be enough to mark the correctness of your encoding/decoding mechanism since it is an official Google implementation. Still, is there any need to test with other languages as well?

Reply:
I think checking just Python is OK. It would present problems only if there are bugs in the Python version that are not also present in other implementations.